recipe/meta.yaml

```diff
@@ -19,16 +19,19 @@ requirements:
   host:
     - pip
     - python
-    - setuptools >=40.6.0
+    - setuptools
     - setuptools_scm >=3.4
     - toml
     - wheel
+    - hatchling
+    - hatch-vcs
+    - pkgutil-resolve-name
```

Comment:
pkgutil-resolve-name should be in the run section, for versions lower than 3.9
https://github.com/python-jsonschema/jsonschema/blob/v4.9.1/pyproject.toml#L43

Reply:
Just made those amendments thanks.